### ISSUE TITLE
Davs doors with java.lang.NullPointerException: null log messages

### ISSUE THREAD

**Issue by cfgamboa** [dCache/dcache#7689]
Dear all,

The following errors are observed on DAVs doors logs. dCache version of door 9.2.17, java 11

Could you please advise?

```
06 Nov 2024 06:50:12 (WebDAV2-dcdoor21-externalipv6) [] 
java.lang.NullPointerException: null
	at org.eclipse.jetty.io.AbstractConnection.lambda$failedCallback$0(AbstractConnection.java:92)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
	at java.base/java.lang.Thread.run(Thread.java:829)
06 Nov 2024 07:10:24 (WebDAV2-dcdoor21-externalipv6) [door:WebDAV2-dcdoor21-externalipv6@webdav2-dcdoor21_httpsDomain:AAYmPVUzvrg] IOException writing to output, probably client terminated connection
06 Nov 2024 07:33:37 (WebDAV2-dcdoor21-externalipv6) [] 
java.lang.NullPointerException: null
	at org.eclipse.jetty.io.AbstractConnection.lambda$failedCallback$0(AbstractConnection.java:92)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

Thank you very much,
Carlos

**Comment by kofemann** [comment]
Hi @cfgamboa,

The issue should be fixed in 9.2.24

https://www.dcache.org/old/downloads/1.9/release-notes-9.2.shtml#release9.2.24
https://github.com/dcache/dcache/commit/09e5a499378e0b3f1f4f159103970351dc540cb9

**Comment by cfgamboa** [comment]
Hello

Thank you very much. We will look at it.

All the best,
Carlos
